extensions/wikia/FounderEmails/events/FounderEmailsCompleteDigestEvent.class.php

```diff
@@ -6,17 +6,19 @@ public function __construct( Array $data = array() ) {
 		$this->setData( $data );
 	}
 
-	public function enabled ( $wikiId, User $user ) {
-		if ( self::isAnswersWiki() ) {
+	public function enabled( User $admin, $wikiId = null ) {
```

Comment:
One note here; in addition to swapping the position of the parameter as noted above, I also changed the variable name to ```$admin``` to make it clear we were acting on an admin user receiving the email rather than the author of the edit/update that triggered the email.  This change can be seen throughout this file.